maven-core/src/main/java/org/apache/maven/project/ProjectModelResolver.java

```diff
@@ -19,14 +19,6 @@
  * under the License.
  */
 
-import java.io.File;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.Iterator;
-import java.util.List;
-import java.util.Set;
-
 import org.apache.maven.model.Dependency;
```

Comment:
Is there a reason why you reformat this?

Reply:
my bad, I might have pressed "optimize import" to make sure no unused import is there.
It resulted in this ordering due to IntelliJ default config.
Should I change it back.

Reply:
No problem. Happens from time to time..and yes it would be great to have the original so the change is better visible...